e3dc/integrationplugine3dc.json

```diff
@@ -0,0 +1,668 @@
+    "vendors": [
+        {
+            "name": "e3dc",
+            "displayName": "E3DC",
+            "id": "59a38851-7f69-4fd8-b01c-c79abf54debf",
+            "thingClasses": [
+                {
+                    "name": "e3dcConnection",
+                    "displayName": "e3dc gateway",
+                    "id": "c05722ec-442f-4feb-8889-39ae3769d068",
+                    "createMethods": ["user", "discovery"],
+                    "interfaces": ["gateway"],
+                    "providedInterfaces": ["solarinverter", "energymeter", "energystorage"],
+                    "paramTypes": [
+                        {
+                            "id": "e9a474c6-7304-424f-9730-8dffd21188d9",
+                            "name":"ipAddress",
+                            "displayName": "IP address",
+                            "type": "QString",
+                            "defaultValue": "127.0.0.1"
+                        },
+                        {
+                            "id": "1d1768c1-421c-4486-9843-22e4646668b7",
+                            "name":"port",
+                            "displayName": "Port",
+                            "type": "int",
+                            "defaultValue": 502
+                        },
+                        {
+                            "id": "371f286f-d14f-48b8-99ba-8d547c8148c9",
+                            "name":"macAddress",
+                            "displayName": "MAC address",
+                            "type": "QString",
+                            "defaultValue": "00:00:00:00:00:00"
+                        },
+                        {
+                            "id": "ac61f55c-4bc4-4815-9a3e-8e01ad2da739",
+                            "name":"slaveId",
+                            "displayName": "Slave ID",
+                            "type": "int",
+                            "defaultValue": 1
+                        }
+                    ],
+                    "stateTypes":[
+                        {
+                            "id": "1596bc6a-69b2-417b-8896-997d2029e1e6",
+                            "name": "connected",
+                            "displayName": "Connected",
+                            "displayNameEvent": "Connected changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "cached": false
+                        }
+                    ]
+                },
+
+                {
+                    "name": "e3dcSmartmeter",
+                    "displayName": "E3dcSmartmeter",
+                    "id": "9ea8830b-3c3f-4ed0-9012-9e7bcd1c25ee",
+                    "createMethods": ["Auto"],
+                    "interfaces": ["energymeter","connectable"],
+                    "paramTypes": [
+                    {
+                        "id": "94904c50-f1fc-41ac-82e0-3bf42334c48c",
+                        "name":"modelId",
```

Comment:
I don't see the modelId actually being used in the code. Is this supposed to be informational to the user?